README.dotnet.md

```diff
@@ -0,0 +1,372 @@
+# 🌟 Azure MCP Server
```

Comment:
is there a reason we need to duplicate the original REAME?

Reply:
Not _strictly_, but since all of the MCP server links contained in it (and the setup instructions, etc) are all NPM-specific I thought it might be useful to show what a `dnx`-facing version might look like. Right now this version only ever seen when users would view the package on nuget.org. There are other options here as well, ranging from `dnx`-subsections in your current README, to little node/.NET clickable links on your install table, to not doing anything at all. That's in my opinion all up to your team how you'd like to present it/move forward.